packages/ui/components/editor/plugins/ToolbarPlugin.tsx

```diff
@@ -220,51 +227,13 @@ function getSelectedNode(selection: RangeSelection) {
   }
 }
 
-type BlockOptionsDropdownProps = {
-  editor: LexicalEditor;
-  blockType: string;
-  toolbarRef: RefObject<HTMLDivElement>;
-  setShowBlockOptionsDropDown: Dispatch<SetStateAction<boolean>>;
-};
-
-function BlockOptionsDropdownList({
```

Comment:
Removing the BlockOptionsDropdownList as we are now using our `Dropdown` component